lib/livebook_web/live/session_live.ex

```diff
@@ -77,6 +78,16 @@ defmodule LivebookWeb.SessionLive do
     end)
   end
 
+  defp assign_policy(socket) do
+    policy =
+      case socket.assigns.live_action do
+        :shared_page -> %{read: true, execute: true, comment: true, edit: true}
+        _ -> %{read: false, execute: false, comment: false, edit: false}
```

Comment:
Shouldn't it be this?

```suggestion
        :shared_page -> %{read: true, execute: false, comment: false, edit: false}
        _ -> %{read: true, execute: true, comment: true, edit: true}
```

Reply:
Yes. So I fixed it. [code](https://github.com/livebook-dev/livebook/pull/1128/files#diff-255c2c5ca297bc981ca82b0e7a6957491bb000b1a0a265c2f43493b5e0c2c03bR1693)